src/App.js

```diff
@@ -1,28 +1,9 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/calculator';
 
 function App() {
```

Comment:
- [ ] Please make the `App.js` component class-based. This way, you move a step forward towards project approval.